sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -21,6 +21,8 @@
 - Changed return type of `BlobClient::GetProperties` to `BlobProperties`.
 - Changed return type of `BlobClinet::GetTags` to `std::map<std::string, std::string>`.
```

Comment:
```suggestion
- Changed return type of `BlobClinet::GetTags` to `std::map<std::string, std::string>`.
- Changed return type of `BlobContainerClient::GetAccessPolicy` to `BlobContainerAccessPolicy`.
```

Reply:
You'd want to do this or the other client library `AccessPolicy` changelogs too.

Reply:
I'll append another commit once LiveTest's finished. Then I don't need to trigger LiveTest again.